domenetjenester/mottak/src/main/java/no/nav/foreldrepenger/mottak/sakskompleks/BerørtBehandlingKontroller.java

```diff
@@ -99,19 +107,35 @@ public class BerørtBehandlingKontroller {
                 sistVedtatteBehandlingForMedforelder.get().getId());
             if (skalBerørtBehandlingOpprettes.isPresent()) {
                 opprettBerørtBehandling(fagsakMedforelder, vedtattBehandlingsresultat, skalBerørtBehandlingOpprettes.get());
+            } else if (skalEndreDekningsgradForMedForelder(vedtattBehandling, sistVedtatteBehandlingForMedforelder.get())) {
```

Comment:
Hm, vil ikke skalBerørtBehandlingOpprettes si true også dersom skalEndreDekningsgrad git logikken i BerørtBehandlingTjeneste? Får du ikke da en berørt i stedet for en endre dekningsgrad?

Reply:
Skulle if (skalEndreDekningsgradForMedForelder vært i blokken under skalBerørtBehandlingOpprettes,isPresent()) ?

Reply:
Evt skulle man sjekket på BerørtÅrsak som kommer fra skalBerørtBehandlingOpprettes?

Reply:
skalBerørtBehandlingOpprettes returnerer empty hvis det er endret dekningsgrad, la inn egen sjekk på det. Usikker på mest oversiktlige måten her